scripts/pi-hole/js/footer.js

```diff
@@ -183,6 +183,13 @@ $(document).keypress(function(e) {
     }
 });
 
+// Submit the login form directly when the remember me checkbox was checked
+$("#logincookie").on("change", function() {
```

Comment:
Why do you set the `on-change` trigger on the check box? I find this behavior - clicking the box automatically submits the entire form - a bit .. unexpected.

Reply:
I thought many users will benefit of that, when they store their credentials in Chrome e.g.
In Nextcloud that's the implementation: https://github.com/nextcloud/server/blob/381967f35e11bfd04baa5c085f733793fee30858/core/js/login.js#L28-L34

Reply:
Hmm, I see. Setting the trigger on the password field itself won't work as it would try the login while the user is typing. So this feature relies on that the browser does not only store the password but also remembers that the user wants to have the checkbox clicked. I'm just asking this as I'm familiar with a few password managers that will not be able to click the box for you (e.g., LastPass). Nevertheless, if you're willing to trust your browser intern password manager, this may work as expected.